### PR TITLE
Update charter to reflect projects current status

### LINKED ISSUE #334: Review project goals
Now that most goals listed in PROJECT.md have been handled at least to some degree, I think it would be beneficial if our project management (at least @brylie and @taija) reviewed which goals have been completed and should some new goals be set.

### PR DESCRIPTION
**Pull request by xylix**
Fixes #334 
Was created in meeting with me, @brylie and @bajiat

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
 ## Risks
-* Duplication of effort
 * Misalignment with business or community goals
 * Failure to execute or prioritize properly
 * Miscommunication of project intent or scope
-* Going into competition with other companies and solutions
 
 ## Value proposition
-User can create an OpenAPI document without knowing the specification or being familiar with JSON or YAML. User is guided by user interface elements such as hints in form fields and mandatory fields.
+User can create and edit OpenAPI documents without knowing the specification or being familiar with JSON or YAML. User is guided by user interface elements such as hints in form fields and mandatory fields and by the sequence of the form.
```
Comment (brylie) [inline CHARTER.md]: > by the sequence of the form.

Consider rewording as:

User is guided by user interface elements, such as hints in form fields and mandatory fields, as well as the user flow. 

User flow means more than 'the order of the field', and can include multi-page editing experience. This also means that the views can be reordered, so they do not so closely align with the OpenAPI specification.

File: CHARTER.md [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
-* User can create a valid OpenAPI document according to OpenAPI specification version 2.0, maybe 3.0.
-* Input validation for the form fields
-* User can download the created document.
-* User can import an existing document to be edited.
-* The user can upload the document to backend service.
-* Communication to storage backend with REST API(s)
+* Experimental support for OpenAPI specification version 3.0
+* Info texts where needed, needs should be determined via testing
+* Support editing existing documents on a server
+* Support better UI integration with platforms where the editor is in an iframe
```
Comment (brylie) [inline CHARTER.md]: Support better UI integration with other platforms, such as packaging on NPM, Bower, or as a Custom Element

**Comment by brylie** [comment]
We added a couple of comments.

**Comment by xylix** [comment]
Changed the lines based on the feedback.

**Pull request merged**